core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/branch/compare/BranchCompareResult.java

```diff
@@ -79,7 +79,7 @@ public Builder totalDeleted(int totalDeleted) {
 			this.totalDeleted = totalDeleted;
 			return this;
 		}
-		
+				
 		public BranchCompareResult build() {
```

Comment:
I think you can safely remove the `build(Set changedContainers)` method and keep the zero-arg build method as the one that actually builds the result.